conversion/conversion.go

```diff
@@ -445,26 +440,60 @@ func UpdateDDLForeignKeys(project, instance, dbName string, conv *internal.Conv,
 	}
 	msg := fmt.Sprintf("Updating schema of database %s in instance %s with foreign key constraints ...", dbName, instance)
 	p := internal.NewProgress(int64(len(fkStmts)), msg, internal.Verbose())
-	for i, fkStmt := range fkStmts {
-		// TODO: Improve performance of the foreign key constraint updates.
-		// For example, issue all of the update ops first before waiting for them to complete
-		// so that that can execute in parallel. We could also print out ids of the
-		// long-running operations.
```

Comment:
We cannot link our internal docs here, thus suggest writing out a short paragraph (
5-6 lines) here explaining the reasoning for the given approach.